src/stellar/stellarEngine.js

```diff
@@ -522,7 +527,7 @@ export class StellarEngine extends CurrencyEngine {
       if (!transaction) {
         throw new Error('ErrorInvalidTransaction')
       }
-      this.log('Broadcasting...')
+      this.log.warn('Broadcasting...')
```

Comment:
Add info about the transaction being broadcast. ie, amount, txId, destination address